src/ast/DECKS.ts

```diff
@@ -1,3 +1,15 @@
 import NODE from "./NODE";
 
-export default class DECKS extends NODE {}
+export default class DECKS extends NODE {
+  decks: string[] = [];
+
+  parse() {
+    this.tokenizer.getAndCheckToken("Decks:");
+    while (this.tokenizer.moreTokens()) {
+      this.decks.push(this.tokenizer.getNext());
+      if (this.tokenizer.checkNext() === ",") {
+        this.tokenizer.getNext();
+      }
+    }
```

Comment:
To handle cases where user's input is one of: Decks: Japanese ,, French | Decks: ,Japanese
we should filter out the commas
```suggestion
    while (this.tokenizer.moreTokens()) {
      this.decks.push(this.tokenizer.getNext());
    }
    decks.filter((deck) => {
       return deck !== ",";
    });
```

Reply:
I don't think that's an issue that we should be concerned with right now, since we're trying to follow the EBNF syntax that we laid out. If we find this an issue later on, we can always change this. But until then, I'm just going to leave this as it is.